tests/features/search/faceting.feature

```diff
@@ -11,14 +11,31 @@ Feature: Search faceting
 
     When I visit `/search?query=`
     And I check the "IMAGE" checkbox
-    And I wait 1 second
+    And I wait 2 seconds
     Then I should be on `/search?query=&page=1&qf=TYPE%3AIMAGE`
     And I see a `filter badge` with the text "Type of media: IMAGE"
 
   Scenario: Filtering results by reusability
 
     When I visit `/search?query=`
     And I check the "open" checkbox
-    And I wait 1 second
+    And I wait 2 seconds
     Then I should be on `/search?query=&page=1&reusability=open`
     And I see a `filter badge` with the text "Can I reuse this?: open"
+
+  Scenario: Filtering results by country
+
+    When I visit `/search?query=`
+    And I check the "Belgium" checkbox
+    And I wait 2 seconds
+    Then I should be on `/search?query=&page=1&qf=COUNTRY%3ABelgium`
+    And I see a `filter badge` with the text "Country: Belgium"
+
+  Scenario: Filtering results by two countries
+
+    When I visit `/search?query=`
+    And I check the "Belgium" checkbox 
+    And I check the "Germany" checkbox
+    And I wait 2 seconds
+    Then I should be on `/search?query=&page=1&qf=COUNTRY%3ABelgium&qf=COUNTRY%3AGermany`
+    And I should have 2 `filter badge`
```

Comment:
let's add some more faceting Scenarios now that we have a couple of facets available:
```
  Scenario: Filtering using a combination of facet fields

    When I visit the `/search?query=`
    And I check the "Belgium" checkbox
    And I check the "IMAGE" checkbox
    And I check the "open" checkbox
    And I wait 2 seconds
    Then I should be on `/search?query=&page=1&reusability=open&qf=COUNTRY%3ABelgium&qf=TYPE%3AIMAGE`
    And I should have 3 `filter badge`s

  Scenario: Facets are loaded from the URL

    When I visit `/search?query=&page=1&reusability=open&qf=COUNTRY%3ABelgium&qf=TYPE%3AIMAGE`
    Then I should have 3 `filter badge`s

  Scenario: Unselecting facets

    When I visit `/search?query=&page=1&reusability=open&qf=COUNTRY%3ABelgium&qf=TYPE%3AIMAGE`
    And I check the "Belgium" checkbox
    And I check the "IMAGE" checkbox
    And I check the "open" checkbox
    And I wait 2 seconds
    Then I should be on the `first page of results`
    And I can't see a `/search?query=`
```